app/csv_exporters/student_profile_csv_exporter.rb

```diff
@@ -32,7 +32,7 @@ def mcas_mathematics_results
   end
 
   def mcas_ela_results
-    student.mcas_ela_results.map do |results|
+    student.mcas_ela_results.map do |result|
```

Comment:
This came up in fixing the export button - these were latent syntax bugs that just hadn't been exercised in the RSpecs.